app/index.html

```diff
@@ -11,18 +11,109 @@
     <!-- endbuild -->
   </head>
   <body class="l-flex-body">
-    <div id="container" class="row">
-      <div class="row-inner">
-        <h1>Chronicle</h1>
-        <h2>Welcome home</h2>
-        <a href="/auth/login">Sign in</a>
-      </div>
+
+  <div class="notification static is-toggle">
+    <div class="row-inner">
+      <p>Chronicle is currently in development. <a href="http://mozillachronicle.tumblr.com/">Learn more.</a></p>
     </div>
-    <footer id="global-footer" class="row">
-      <div class="row-inner">
-        <p>&copy; 2015 &bull; <a href="http://mozilla.org/" target="_blank">Mozilla</a></p>
+  </div>
+
+    <div id="landing-page-wrapper">
+    <div id="landing-page">
+      <div id="cover" class="section">
+        <div class="l-flex-vertical-centered">
+          <div class="content">
+            <div class="logo"></div>
```

Comment:
I'm seeing this in Chrome (no logo or anything in the circle):

![chronicle](https://cloud.githubusercontent.com/assets/557895/6193834/d5fd9f70-b373-11e4-8c7d-e148c5c0f6fa.png)